scripts/pre_leap.sh

```diff
@@ -19,6 +19,11 @@
 set -e -u
 set -o pipefail
 
+# Pre-flight check
+pushd /opt/rpc-upgrades/playbooks
```

Comment:
Let's move this to somewhere like here:

https://github.com/rcbops/rpc-upgrades/blob/af342f588a08000dad1a4ec6bbf6aabf3b04db1f/scripts/ubuntu14-leapfrog.sh#L65

So that it runs before any modifications to the system occur.

Reply:
Cool, earlier is better